autoconfigure/storage-stackdriver/src/main/java/zipkin/autoconfigure/storage/stackdriver/ZipkinStackdriverStorageAutoConfiguration.java

```diff
@@ -20,6 +20,7 @@
 import io.grpc.ManagedChannelBuilder;
 import io.grpc.auth.MoreCallCredentials;
 import io.netty.handler.ssl.OpenSsl;
+import io.netty.util.internal.PlatformDependent;
```

Comment:
not concerned on the internal dep as the only consumer of auto-config is the server module (ex this won't taint the deps of those using zipkin-reporter senders)